src/schema/sale/index.js

```diff
@@ -101,6 +101,39 @@ export const SaleType = new GraphQLObjectType({
           return fetch.then(invert).then(exclude(options.exclude, "id"))
         },
       },
+      artworksConnection: {
+        type: artworkConnection,
+        description: "Returns a connection of artworks for a sale.",
+        args: pageable({
+          exclude: {
+            type: new GraphQLList(GraphQLString),
+            description:
+              "List of artwork IDs to exclude from the response (irrespective of size)",
+          },
+        }),
+        resolve: (
+          { eligible_sale_artworks_count, id },
+          options,
+          _request,
+          { rootValue: { accessToken, saleArtworksLoader } }
+        ) => {
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            options
+          )
+          const gravityArgs = { page, size }
+          const invert = saleArtworks => map(saleArtworks, "artwork")
+          return saleArtworksLoader(id, options)
+            .then(({ body }) => body)
+            .then(invert)
+            .then(exclude(options.exclude, "id"))
```

Comment:
@mzikherman - you sure this is ok? Wont this break pagination?

Reply:
Yea, I think the way we're using this we just want _a_ page of results (and if that is one off b/c the given work is included), that's fine.

But this would be the case of trying to do some filtering _after_ a true backend pagination, which is bound to lead to some issue somewhere.

Really, Gravity/backend APIs should be offering a proper 'exclude ids' functionality that can just accept an `exclude_ids` query param (this is already supported in a number of API V1 endpoints), and 'properly' exclude them, while also properly paginating.

Looking at the corresponding endpoint in Grav: https://github.com/artsy/gravity/blob/7e135fc52178b174907bf540f16191af5bc58ca3/app/api/v1/sales_artworks_endpoint.rb#L70 , that should be fairly easy to add. Would you like to do that/pair?

Otherwise, we can leave this for now, and once that Gravity PR is in we can add that param here/remove this line.

Reply:
I think the fix is in these two lines: https://github.com/artsy/gravity/blob/7e135fc52178b174907bf540f16191af5bc58ca3/app/api/v1/sales_artworks_endpoint.rb#L83-L84

Basically, we have an array of artwork id's that we use to form the basis for looking up sale artworks...so we can just remove any `exclude_ids` from that. One-liner!

Reply:
Sounds good, I'll tackle that real quick.